twitter/tweetarchiver/archiver.py

```diff
@@ -62,16 +62,15 @@ def close(self):
             self.f = None
             try:
                 os.rename(self.fn + '.open', self.fn)
-            except Exception, ex:
+            except Exception as ex:
                 logging.warn('failed to rename %s.open to %s (%s)',
                              self.fn, self.fn, ex)
     def write_record(self, message):
         """message must be one whole streaming message."""
         if self.f is None:
-            raise IOError, "attempted to write into closed file %s" % self.fn
+            raise IOError("attempted to write into closed file %s" % self.fn)
         z = GzipFile(fileobj=self.f, mode='wb', compresslevel=self.complevel)
-        z.write(message)
-        z.write('\r\n')
+        z.write(message.encode()+b'\r\n')
```

Comment:
it is a good practice to specify encoding explicitly, rather than replying on the system default.

Reply:
Thank you. I have specified it now